File: Iverson.Clients/Go/sample/models/tag.go
package models

// Tag is a sample entity demonstrating a many-to-many relation.
type Tag struct {
	Id       string   `iverson_key:"true" iverson_guid:"true"`
	Name     string   `iverson_search_key:"0"`
	Articles []string `iverson:"many_to_many:Article"`
	TenantId string   `iverson_tenant:"true"`
}

File: Iverson.Clients/TypeScript/sample/models/Tag.ts
import 'reflect-metadata';
import { IversonEntity, IversonGuid, IversonKey } from '../../src/annotations.js';

@IversonEntity()
export class Tag {
    @IversonKey()
    @IversonGuid()
    id: string = '';

    label: string = '';
}
